### PR TITLE
Update jo.m3u

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: channels/jo.m3u [hunk 1]
```diff
@@ -15,8 +15,28 @@ https://jrtv-live.ercdn.net/jordanhd/jordanhd.m3u8
 https://cdn3.wowza.com/1/ZFBldUlPNjRBRDZM/ZW90V2ZW/hls/live/playlist.m3u8
 #EXTINF:-1 tvg-id="NojoumTV.jo" tvg-country="JO" tvg-language="Arabic" tvg-logo="https://i.imgur.com/6ttWNu0.jpg" group-title="Music",Nojoum TV (720p) [Offline]
 http://51.210.199.41/hls/stream.m3u8
-#EXTINF:-1 tvg-id="RotanaRadio.jo" tvg-country="JO" tvg-language="Arabic" tvg-logo="http://service.mobile.rotanaradio.jo/sites/default/files/studio.png" group-title="Music",Rotana Radio (220p) [Not 24/7]
+#EXTINF:-1 tvg-id="" tvg-country="JO" tvg-language="Arabic" tvg-logo="" group-title="Religious",Radio Yaqeen Jordan: Control Studio
+http://188.247.86.66/MixFM/ControlStudio1/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/jo.m3u:19]: Not 24/7 I shall assume. Got nothing right now.

Comment (BathMubashar) [reply channels/jo.m3u:19]: Fixed

File: channels/jo.m3u [hunk 2]
```diff
@@ -15,8 +15,28 @@ https://jrtv-live.ercdn.net/jordanhd/jordanhd.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-country="JO" tvg-language="Arabic" tvg-logo="" group-title="Religious",Radio Yaqeen Jordan: Control Studio [Not 24/7]
+http://188.247.86.66/MixFM/ControlStudio1/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="JO" tvg-language="Arabic" tvg-logo="" group-title="Religious",Radio Yaqeen Jordan: Studio
+http://188.247.86.66/MixFM/OnAirStudio1/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="JO" tvg-language="Arabic" tvg-logo="" group-title="Religious",Radio Yaqeen Jordan: City View
+http://188.247.86.66/MixFM/CityView1/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="JO" tvg-language="Arabic" tvg-logo="" group-title="Music",Rotana Tarab Jordan: Control Studio
+http://188.247.86.66/RotanaRadio/Audio32/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="JO" tvg-language="Arabic" tvg-logo="" group-title="Music",Rotana Tarab Jordan: City View
+http://188.247.86.66/RadioFann/Audio32/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/jo.m3u:27]: Are you sure this one is Rotana Tarab ? It's got the Fann mention.

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]
See messages.

File: channels/jo.m3u (hunk 2 above)
Comment (BathMubashar) [reply channels/jo.m3u:27]: Yes, the logo on the top right and sound is identical to

#EXTINF:-1 tvg-id="" tvg-country="JO" tvg-language="Arabic" tvg-logo="" group-title="Music",Rotana Tarab Jordan: Control Studio
http://188.247.86.66/RotanaRadio/Audio32/playlist.m3u8

They re-used the url

**Review by BellezaEmporium — APPROVED** [review]

**Comment by BathMubashar** [comment]
Thanks resolved

**Review by jkemp814 — APPROVED** [review]

**Pull request merged**